### PR TITLE
add sub-stream benchmark

### PR DESCRIPTION
**Pull request by patriknw**
* and remove invalid graph_with_nested_imports

### CODE REVIEW

**Comment by patriknw** [comment]
408593.165 ± 11501.751   ops/s

**Comment by ktoso** [comment]
LGTM 👍

File: akka-bench-jmh/src/main/scala/akka/stream/MaterializationBenchmark.scala [hunk 1]
```diff
@@ -68,6 +56,24 @@ object MaterializationBenchmark {
       out ~> Sink.ignore
       ClosedShape
     })
+
+  final val subStreamCount = 100000
```
Comment (patriknw) [inline akka-bench-jmh/src/main/scala/akka/stream/MaterializationBenchmark.scala]: perhaps this is too much for each iteration, I'll try with smaller

**Review by johanandren — APPROVED** [review]
LGTM

**Comment by patriknw** [comment]
I'm changing the jenkins jobs

**Comment by patriknw** [comment]
ah, it should just work, nice

disabling akka-stream-perf-tol

**Pull request merged**